charts/sentry-apps/templates/application.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
```

Comment:
The argoconfigs library chart has a named helper to make generating argo apps less repetitive.

Reply:
How would you use it?